tests/models/flaubert/test_modeling_flaubert.py

```diff
@@ -431,6 +432,13 @@ def test_flaubert_model(self):
         config_and_inputs = self.model_tester.prepare_config_and_inputs()
         self.model_tester.create_and_check_flaubert_model(*config_and_inputs)
 
+    def test_flaubert_model_with_sinusoidal_encodings(self):
```

Comment:
let's use copied from for the test as well!

Reply:
does this solve it?